resources/js/visual-editor/site-editor/styles/panels/panel-controls.css

```
@@ -0,0 +1,134 @@
+    flex-wrap: wrap;
+    gap: 0.25rem 0.5rem;
+    justify-content: flex-start;
+    min-width: 0;
+}
+
+.ap-site-editor__style-control-row-base {
+    font-size: 0.75rem;
+    margin: 0;
+    opacity: 0.75;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)